dvc/commands/data.py

```diff
@@ -0,0 +1,178 @@
+import argparse
+import logging
+from typing import TYPE_CHECKING
+
+from funcy import compact, log_durations
+
+from dvc.cli.command import CmdBase
+from dvc.cli.utils import append_doc_link, fix_subparsers
+from dvc.ui import ui
+
+if TYPE_CHECKING:
+    from dvc.repo.data import Status as DataStatus
+
+
+logger = logging.getLogger(__name__)
+
+
+class CmdDataStatus(CmdBase):
+    COLORS = {
+        "not_in_cache": "red",
+        "committed": "green",
+        "uncommitted": "yellow",
+        "untracked": "cyan",
+    }
+    LABELS = {
+        "not_in_cache": "Not in cache",
+        "committed": "DVC committed changes",
+        "uncommitted": "DVC uncommitted changes",
+        "untracked": "Untracked files",
+        "unchanged": "DVC unchanged files",
+    }
+    HINTS = {
+        "not_in_cache": 'use "dvc pull <file>..." '
+        "to update your local storage",
+        "committed": "git commit the corresponding dvc files "
+        "to update the repo",
```

Comment:
```suggestion
        "committed": "`git commit` the corresponding DVC metafiles "
        "to update the repo.",
```

Reply:
I prefer quotes, it's easier on eyes. :)

Reply:
I guess it's a convention already to code-inline this kind of things?

Reply:
We do use backticks on help messages, but I am not sure about the reasoning. The only other place we print commands is in `git add` hints where we just indent those.

Reply:
My only rec. here is to aim for consistency if possible. Probably not major (up to you) but it's a product quality question.